buildspec-e2e-release-pr-join-artifacts.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.2
+
+phases:
+
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR
+      - git checkout main
```

Comment:
The reason I am checking out `main` is to run tests from `main` branch. I think `main` is more accurate than `rc` because `rc` may have changed. What is your thoughts? @ilfa

Reply:
that also makes sense. The concern is that, `rc` branch may have changed the test files, so it made sense to use test files from `main`. On the other hand, we don't test the test files from `rc` in the current approach until it is merged. After the merge, next release will be tested with these new test files.

Let's use `rc`, it seems more logical.